src/registrar/templates/includes/required_fields.html

```diff
@@ -1,3 +1,3 @@
-  <p class="margin-top-3">
+  <p class="{% if not remove_margin_top %}margin-top-3 {% endif %}">
```

Comment:
Yay for DOM magic... stuff like this makes me think our frontend architecture is ripe for a once-over, but I'm not going to open that can of worms for this ticket.  ROI wouldn't be very high anyways. Good workaround.